app/scripts/helpers/price.helpers.js

```diff
@@ -0,0 +1,3 @@
+export function formatPrice(amount, currency) {
+	return `${currency === 'USD' ? '$' : ''}${amount}${currency === 'EUR' ? ' €' : ''}`;
```

Comment:
I thought about this and we discussed it a little bit on Slack. It really depends on the behavior we want...

I read that in English countries, they write prices before the amount with no space (e.g. €15 or $15) while in France, Italy... it is a non-breaking space + sign (e.g. 15 € or 15 $).

So we've got different options:
1. We follow the spec, leaving the browser and the language to decide how to display it. (a bit weird since everything is in English).
2. We follow the spec, forcing the en-US language. (and we get €15, $15 everywhere).
3. We stay this way using a custom thing.

At least with this helper, we could change our mind and reflect changes easily.

Any preferences @yannickmathey @Aldrian ?